plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -1,104 +1,272 @@
 package io.jenkins.tools.pluginmanager.cli;
 
+import io.jenkins.tools.pluginmanager.config.Config;
 import io.jenkins.tools.pluginmanager.config.Settings;
+import io.jenkins.tools.pluginmanager.impl.Plugin;
 import java.io.File;
+import java.io.FileNotFoundException;
+import java.net.MalformedURLException;
 import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
+import org.apache.commons.lang3.StringUtils;
 import org.kohsuke.args4j.Option;
 import org.kohsuke.args4j.spi.BooleanOptionHandler;
 import org.kohsuke.args4j.spi.FileOptionHandler;
 import org.kohsuke.args4j.spi.StringArrayOptionHandler;
 import org.kohsuke.args4j.spi.URLOptionHandler;
 
+import static java.util.stream.Collectors.toList;
 
-public class CliOptions {
+class CliOptions {
     //path must include plugins.txt
     @Option(name = "--plugin-file", aliases = {"-f"}, usage = "Path to plugins.txt file",
             handler = FileOptionHandler.class)
-    public File pluginTxt;
+    private File pluginTxt;
 
     @Option(name = "--plugin-download-directory", aliases = {"-d"},
             usage = "Path to directory in which to install plugins",
             handler = FileOptionHandler.class)
-    public File pluginDir;
+    private File pluginDir;
```

Comment:
So I had kind of modeled what I was doing after the custom war packager. Is there a reason why in the CLI for that, all of the CLI variables are public? https://github.com/jenkinsci/custom-war-packager/blob/master/custom-war-packager-cli/src/main/java/io/jenkins/tools/warpackager/cli/CliOptions.java